### PR TITLE
Add option keep model pipeline extras

### PR DESCRIPTION
**Pull request by hpinkos**
cc @ggetz

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @hpinkos!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/pipeline-extras/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/Model.js [hunk 1]
```diff
@@ -654,6 +654,8 @@ define([
         this._rtcCenterEye = undefined; // in eye coordinates
         this._rtcCenter3D = undefined;  // in world coordinates
         this._rtcCenter2D = undefined;  // in projected world coordinates
+
+        this._keepPipelineExtras = options.keepPipelineExtras; // keep the buffers in memory for use in other applications
```
Comment (ggetz) [inline Source/Scene/Model.js]: Default value of `false`?

Comment (hpinkos) [reply Source/Scene/Model.js]: There's no need.  `false` and `undefined` both evaluate to `false` in the if statement.  I'm not sure if we have a coding standard for this for this repo, but on some of the other projects I work on we usually don't bother default valuing to false.

**Comment by ggetz** [comment]
Looks good @hpinkos

**Pull request merged**